CMakeLists.txt

```
@@ -68,9 +68,9 @@ set(OBJECTS_VERSION "1.3.11")
 set(OBJECTS_URL  "https://github.com/OpenRCT2/objects/releases/download/v${OBJECTS_VERSION}/objects.zip")
 set(OBJECTS_SHA1 "02ddf6c685a2da8bac98a57b34be95d63192443d")
 
-set(OPENSFX_VERSION "1.0.3")
+set(OPENSFX_VERSION "1.0.2")
 set(OPENSFX_URL  "https://github.com/OpenRCT2/OpenSoundEffects/releases/download/v${OPENSFX_VERSION}/opensound.zip")
-set(OPENSFX_SHA1 "ffec5a97bd5035860c4c9a43fe32cf51886478e3")
+set(OPENSFX_SHA1 "64EF7E0B7785602C91AEC66F005C035B05A2133B")
```

Looks like an accidental revert.

```suggestion
set(OPENSFX_SHA1 "ffec5a97bd5035860c4c9a43fe32cf51886478e3")
```